binaryen-c.h: this function declaration is not a prototype

When compiling with Binaryen on Emscripten's clang (related to #1494), I get the following warnings:

```
/home/trosinenko/projects/qemujs/build/emscripten_-m32_-Os_-g2/binaryen/src/binaryen-c.h:655:29: warning: this function declaration is not a prototype [-Wstrict-prototypes]
int BinaryenGetOptimizeLevel();
                            ^
                             void
/home/trosinenko/projects/qemujs/build/emscripten_-m32_-Os_-g2/binaryen/src/binaryen-c.h:663:27: warning: this function declaration is not a prototype [-Wstrict-prototypes]
int BinaryenGetShrinkLevel();
                          ^
                           void
/home/trosinenko/projects/qemujs/build/emscripten_-m32_-Os_-g2/binaryen/src/binaryen-c.h:671:25: warning: this function declaration is not a prototype [-Wstrict-prototypes]
int BinaryenGetDebugInfo();
                        ^
                         void
```

Not too severe, but it disturbs from more significant warnings. Sure I would fix it myself, but analyzing CLA is probably order of magnitude harder than actually fixing :)

https://github.com/WebAssembly/binaryen/blob/da3bb2ffcc8a6efd6a95a07f7de793d1362f4a07/src/binaryen-c.h#L655

--- a/src/binaryen-c.cpp
+++ b/src/binaryen-c.cpp
@@ -1946,7 +1946,7 @@ void BinaryenModuleOptimize(BinaryenModuleRef module) {
   passRunner.run();
 }
 
-int BinaryenGetOptimizeLevel() {
+int BinaryenGetOptimizeLevel(void) {
   if (tracing) {
     std::cout << "  BinaryenGetOptimizeLevel();\n";
   }
@@ -1962,7 +1962,7 @@ void BinaryenSetOptimizeLevel(int level) {
   globalPassOptions.optimizeLevel = level;
 }
 
-int BinaryenGetShrinkLevel() {
+int BinaryenGetShrinkLevel(void) {
   if (tracing) {
     std::cout << "  BinaryenGetShrinkLevel();\n";
   }
@@ -1978,7 +1978,7 @@ void BinaryenSetShrinkLevel(int level) {
   globalPassOptions.shrinkLevel = level;
 }
 
-int BinaryenGetDebugInfo() {
+int BinaryenGetDebugInfo(void) {
   if (tracing) {
     std::cout << "  BinaryenGetDebugInfo();\n";
   }

--- a/src/binaryen-c.h
+++ b/src/binaryen-c.h
@@ -652,7 +652,7 @@ void BinaryenModuleOptimize(BinaryenModuleRef module);
 
 // Gets the currently set optimize level. Applies to all modules, globally.
 // 0, 1, 2 correspond to -O0, -O1, -O2 (default), etc.
-int BinaryenGetOptimizeLevel();
+int BinaryenGetOptimizeLevel(void);
 
 // Sets the optimization level to use. Applies to all modules, globally.
 // 0, 1, 2 correspond to -O0, -O1, -O2 (default), etc.
@@ -660,7 +660,7 @@ void BinaryenSetOptimizeLevel(int level);
 
 // Gets the currently set shrink level. Applies to all modules, globally.
 // 0, 1, 2 correspond to -O0, -Os (default), -Oz.
-int BinaryenGetShrinkLevel();
+int BinaryenGetShrinkLevel(void);
 
 // Sets the shrink level to use. Applies to all modules, globally.
 // 0, 1, 2 correspond to -O0, -Os (default), -Oz.
@@ -668,7 +668,7 @@ void BinaryenSetShrinkLevel(int level);
 
 // Gets whether generating debug information is currently enabled or not.
 // Applies to all modules, globally.
-int BinaryenGetDebugInfo();
+int BinaryenGetDebugInfo(void);
 
 // Enables or disables debug information in emitted binaries.
 // Applies to all modules, globally.

